test/functorch/test_eager_transforms.py

```diff
@@ -146,6 +149,30 @@ def test_argnums_reorders(self):
         res = _slice_argnums(args, (1, 0))
         self.assertEqual(res, (args[1], args[0]))
 
+def _get_weights_and_functional_call(net, mechanism):
```

Comment:
All of the tests for functional_call seem integration-y (they run on a model).

Could we add some basic API unittests to test things like:
- functional_call with a single dict
- functional_call with multiple dicts
- functional_call with args, kwargs
- functional_call's special case where it accepts a single input

This way, if someone modifies torch.func.functional_call, the tests will notice it

Reply:
Added tests to `test_stateless.py`